data/plugins/projectsearch.lua

```diff
@@ -239,36 +239,36 @@ end
 
 
 command.add(nil, {
-  ["project-search:find"] = function()
-    core.command_view:enter("Find Text In Project", {
+  ["project-search:find"] = function(path)
+    core.command_view:enter("Find Text In " .. (path or "Project"), {
```

Comment:
It might be better to show `path` relative to the current project directory.

Reply:
You are right. I'll fix.

Reply:
OK, I changed this to use paths relative to whichever project directory is the parent project.